Tests-Legacy/L0/SshV0/_suite.ts

```diff
@@ -1,202 +0,0 @@
-/// <reference path="../../definitions/mocha.d.ts"/>
-/// <reference path="../../definitions/node.d.ts"/>
-
-import assert = require('assert');
-import trm = require('../../lib/taskRunner');
-import path = require('path');
-
-function setResponseFile(name: string) {
-    process.env['MOCK_RESPONSES'] = path.join(__dirname, name);
-}
-
-describe('SshV0 Suite', function() {
-    this.timeout(parseInt(process.env.TASK_TEST_TIMEOUT) || 20000);
-
-    before((done) => {
-        // init here
-        done();
-    });
-
-    after(function () {
-
-    });
-    it('Fails for missing endpoint', (done) => {
-        setResponseFile('responseEndpoint.json');
-
-        var tr = new trm.TaskRunner('SshV0', true, true);
-
-        tr.run()
-            .then(() => {
-                assert(tr.invokedToolCount == 0, 'should not have run any tools');
-                assert(tr.resultWasSet, 'task should have set a result');
-                assert(tr.stderr.length > 0, 'should have written to stderr');
-                assert(tr.failed, 'task should have failed');
-                assert(tr.stderr.indexOf('Input required: sshEndpoint') >= 0, 'wrong error message: "' + tr.stderr + '"');
-                done();
-            })
-            .fail((err) => {
-                done(err);
-            });
-    })
-    //it('Fails for invalid private key', (done) => {
```

Comment:
this was commented out, but should it have been?

Reply:
I'm not 100% sure, but its been commented out for the past 3 years, maybe longer. I think its covered by the tests further down (specifically 'Fails when connection cannot be made with given details')